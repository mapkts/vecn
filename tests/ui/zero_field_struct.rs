#[vecn::vector]
struct TupleStruct();

#[vecn::vector]
struct NamedStruct{}

fn main() {}
