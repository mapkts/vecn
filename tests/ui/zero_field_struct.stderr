error: expected at least one field, but found nothing
 --> $DIR/zero_field_struct.rs:2:19
  |
2 | struct TupleStruct();
  |                   ^^

error: expected at least one field, but found nothing
 --> $DIR/zero_field_struct.rs:5:19
  |
5 | struct NamedStruct{}
  |                   ^^
